keyboards/redragon/k556/info.json

```
@@ -0,0 +1,314 @@
+{
+    "keyboard_name": "K556 Devajaras",
+    "manufacturer": "Redragon",
+    "url": "https://www.redragonzone.com/products/redragonk556",
+    "maintainer": "Xinayder",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x5004",
+        "device_version": "1.0.0",
+        "force_nkro": true
```

```suggestion
        "device_version": "1.0.0"
```
is there a reason nkro is enforced by default?

---

I just based off the previous code, which had NKRO enabled. After a quick search, the keyboard does support NKRO. Is it a problem if it's enabled by default?

---

NKRO is supported and enabled, but managed by the host. Enforcing it on by default might introduce issues with dumb hosts like BIOS and apparently even macOS.